llvm/utils/mlgo/mlgo/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
The canonical way to setup the folder structure for a python package is to have the root folder named after the project (I guess it could be different), and then have a subfolder with the package name. So if we used `mlgo/corpus`, the package would be named `corpus`. I think `mlgo-corpus` would be a decent name that would allow extensability in the future with other `mlgo-*` packages?

The lit tests are under `llvm/utils/lit/tests`, similar to how they would be here. I was planning on not wiring everything in with CMake, but it's a good point that the tests should be wired in for that.

I don't think it would be super trivial to get `lit.local.cfg` to read `pyproject.toml` automatically. But given the requirements are pretty sparse (one package, `absl`, and a minimum of Python 3.8), I think explicitly enumerating them shouldn't be a big deal.

Reply:
Or what's probably better for structuring is we can do `mlgo/mlgo/corpus` and then the package would be accessed as `mlgo.corpus` while still keeping everything together if we want to add more in the future. I'll switch to that for now.

Reply:
Looks like all the other Python tests within the monorepo are pretty much lit tests. I'll work on converting these tests to lit tests later today. Should be feasible since we're essentially just using Python tooling to validate where files are, which we can easily do in lit.

Reply:
Ack - thanks for clarifying the directory structure; looking around, there doesn't seem to be much of a precedent either; so... how about `llvm/utils/mlgo-utils/mlgo`? 

(tests moving to lit sounds perfect)